src/ui/MissingTerm/MissingTerms.ts

```diff
@@ -0,0 +1,223 @@
+      });
+    }
+  }
+
+  private buildContainer(): HTMLElement[] {
+    const elements: HTMLElement[] = [];
+    elements.push(this.buildCaption().el);
+    this.buildMissingTerms().forEach(term => {
+      if (term) {
+        elements.push(term.el);
+      }
+    });
+    if (elements.length > 1) {
+      return elements;
+    }
+  }
+
+  private buildCaption(): Dom {
+    return $$('span', { className: 'coveo-field-caption' }, this.options.caption);
+  }
+
+  private buildMissingTerms(): Dom[] {
+    const terms: Dom[] = this.missingTerms.map(term => {
+      if (this.hideMissingTermifFeaturedResults(term) || this.hideMissingTermIfWildcard(term)) {
+        return;
+      }
+      return this.makeTermClickableIfEnabled(term);
+    });
+    return terms;
+  }
+
+  private executeNewQuery(missingTerm: string = this.queryStateModel.get('q')) {
+    this.usageAnalytics.logSearchEvent<IAnalyticsIncludeMissingTerm>(analyticsActionCauseList.missingTermClick, {
+      missingTerm: missingTerm
+    });
+    this.queryController.executeQuery();
+  }
+
+  private makeTermClickableIfEnabled(term: string): Dom {
+    if (this.options.clickable) {
+      const termElement = $$('button', { className: 'coveo-missing-term coveo-clickable' }, term);
+      termElement.on('click', () => {
+        this.addTermForcedToAppear(term);
+        this.executeNewQuery(term);
+      });
+      return termElement;
+    } else {
+      return $$('span', { className: 'coveo-missing-term' }, term);
+    }
+  }
+
+  private createWordBoundryDelimitedRegex(term: string): RegExp {
+    return XRegExp(`${this.wordBoundary}(${term})${this.wordBoundary}`, 'g');
+  }
+
+  private hideMissingTermifFeaturedResults(term: string): boolean {
+    this.updateTermForcedToAppear();
+    return this.termForcedToAppear.indexOf(term) !== -1;
+  }
+
+  private hideMissingTermIfWildcard(term): boolean {
+    const query = this.queryStateModel.get('q');
+    const regxStarWildcard = XRegExp(`(\\*${term})|${term}\\*`);
+    const regxxQuestionMarkWildcard = XRegExp(`(\\?${term})|${term}\\?`);
+
+    const foundStar = this.queryController.getLastQuery().wildcards && regxStarWildcard.test(query);
+    const foundQuestionMark = this.queryController.getLastQuery().questionMark && regxxQuestionMarkWildcard.test(query);
+
+    return foundStar || foundQuestionMark;
+  }
+
+  private hideMissingTermIfNumberOfResult(elements: HTMLElement[]) {
+    const allMissingTerms = elements.filter(element => {
+      return element.tagName === 'BUTTON';
+    });
+    if (allMissingTerms.length <= this.options.numberOfResults) {
+      return;
+    }
+    for (let index = this.options.numberOfResults; index < allMissingTerms.length; index++) {
+      allMissingTerms[index].setAttribute('style', 'display: none');
+    }
+    const showMore = $$(
+      'button',
+      { className: 'coveo-missing-term-show-more coveo-clickable' },
+      `${allMissingTerms.length - this.options.numberOfResults} more...`
```

Comment:
You should localize that string, see the example in the file DynamicFacetBreadcrumbs.ts line 67